core/jvm/src/test/scala/scalaz/zio/RTSSpec.scala

```diff
@@ -1033,6 +1034,15 @@ class RTSSpec(implicit ee: ExecutionEnv) extends TestRuntime {
     } yield value must_=== true
   )
 
+  def testInterruptedBlockingIO = unsafeRun(
+    for {
+      flag  <- Ref.make(false)
+      fiber <- blocking.interruptible { Thread.sleep(Long.MaxValue) }.ensuring(flag.set(true)).fork
+      res   <- fiber.interrupt
+      value <- flag.get
```

Comment:
100% that `ensuring` will execute before `flag.get` ? I'm afraid of test failures.

Reply:
Yes, 100%. If it doesn't, we have a bigger problem.